src/Symfony/Component/Security/Core/Resources/translations/security.th.xlf

```diff
@@ -74,10 +74,6 @@
                 <source>Too many failed login attempts, please try again in %minutes% minute.</source>
                 <target>มีความพยายามเข้าสู่ระบบล้มเหลวมากเกินไป โปรดลองอีกครั้งใน %minutes% นาที</target>
             </trans-unit>
-            <trans-unit id="20">
-                <source>Too many failed login attempts, please try again in %minutes% minutes.</source>
-                <target>มีความพยายามเข้าสู่ระบบล้มเหลวมากเกินไป โปรดลองอีกครั้งใน %minutes% นาที</target>
-            </trans-unit>
```

Comment:
Removing this looks like a mistake. Can you verify?

Reply:
trans-unit 19 and 20 are exactly the same to I removed 20

Reply:
Isn't 19 for singular variant `minute`?